lib/schema-evolution-manager/apply_util.rb

```diff
@@ -21,9 +21,24 @@ def apply!(dir)
       Preconditions.check_state(File.directory?(dir),
                                 "Dir[%s] does not exist" % dir)
 
-      count = 0
+      pending_scripts = []
       @scripts.each_pending(dir) do |filename, path|
-        count += 1
+        pending_scripts << [filename, path]
+      end
+
+      if pending_scripts.size > 1
+        puts "WARNING: You are about to apply more than 1 script:"
```

Comment:
Suggest:
  - Do not output the warning if a dry run - there is no impact on dry run
  - Remove "WARNING" - it is a normal use case (for me at least) to have multiple scripts to apply. Perhaps say "Please confirm that you would like to apply all (3) of the pending scripts"